governance/reviews/cloudevents-08-29-2023.md

```diff
@@ -0,0 +1,210 @@
+
+**SDK Maintainers**: the identified owners of each of the language SDKs.  See Subprojects for more about these.
+
+While the Admins are the "senior" body in CloudEvents, their powers over Voting Members are strictly limited in order to ensure the neutrality of specification votes.
+
+### Discoverability
+
+#### Governance Location
+
+Governance files are all located in the [Documentation directory](https://github.com/cloudevents/spec/blob/main/docs/GOVERNANCE.md) of the primary specification repository.  
+
+#### Governance Discovery Completeness
+
+The Governance files that are present are easily found from the main repository.  The main governance file is also linked from the website menus.  They are also linked from the main spec README.
+
+Governance files are not generally interlinked, nor do they seem to refer to each other at all; it's up to the individual contributor to figure out which files apply to their role.  This could be improved a great deal as part of a general governance cleanup.
+
+Many bits of governance documentation are found in different files than one would conventionally expect them to be found.  Contributor information is found in Governance or Maintainers files; decision making requirements in Contributors files.
+
+### Documentation Content
+
+<!--- Provide the commit of the file under evaluation as a point-in-time reference to this review. --->
+
+The following table details the governance areas expected for a project. Coverage is indicated by Complete, Partial, and Missing.
+* Complete - the content of the governance documentation is fully detailed and does not leave any question to the reader.
+* Partial - the content of the governance documentation is missing some information and would leave the reader with questions or some level of misunderstanding.
+* Missing - the documentation is absent, wholly undiscoverable, or woefully inadequate in meeting the objectives of that governance content. The reader cannot act on the content that is available.
+* Unknown - status cannot be assessed at this time
+
+| Governance Area | Coverage | Documents | Finding Notes |
+|:----------------|:--------:|:------:|:--------------|
+| Project Purpose | Complete | [README](https://github.com/cloudevents/spec) | |
+| Maintainer List | Partial | [OWNERS](https://github.com/cloudevents/spec/blob/main/OWNERS), [Spreadsheet](https://docs.google.com/spreadsheets/d/1bw5s9sC2ggYyAiGJHEk7xm-q2KG6jyrfBy69ifkdmt0/edit?pli=1#gid=0) | See discussion above |
+| Code of Conduct | Complete | Uses link to CNCF CoC | |
+| Contributor Guide | Partial | [CONTRIBUTING](https://github.com/cloudevents/spec/blob/main/docs/CONTRIBUTING.md) | See discussion above |
+| Contributor Ladder | Partial | [GOVERNANCE](https://github.com/cloudevents/spec/blob/main/docs/GOVERNANCE.md) | Several roles exist, but qualifications and advancement are not explained |
```

Comment:
Can you elaborate on what might be missing from the #membership section of the GOVERNANCE.md doc?